package.json

```diff
@@ -21,10 +21,12 @@
     "email": "julien.fontanet@isonoe.net"
   },
   "preferGlobal": false,
-  "main": "dist/",
+  "main": "dist/index.js",
```

Comment:
How is it related?

Reply:
It's unrelated. I just want to make it explicate for the main entry.

Sorry for have not to wait for your review of this PR because I think the changes are trivial.

Please let me know if you want me to remove this change and I can send another PR to revert this change.